docs/reference/commandline/node_ls.md

```diff
@@ -49,48 +49,76 @@ than one filter, then pass multiple flags (e.g., `--filter "foo=bar" --filter "b
 
 The currently supported filters are:
 
-* name
-* id
-* label
+* [id](#ID)
+* [label](#Label)
+* [membership](#Membership)
+* [name](#Name)
+* [role](#Role)
```

Comment:
oh! just recalled that in the new docs system, anchor tags need to be prepended with the document name (yup, it's annoying), so;

```Markdown
* [id](node_ls.md#ID)
* [label](node_ls.md#Label)
* [membership](node_ls.md#Membership)
* [name](node_ls.md#Name)
* [role](node_ls.md#Role)
```